src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resultObject = {};
+  const splitString = sourceString.split(';');
+
+  splitString.forEach((element) => {
+    const splitElement = element.split(':').map((item) => item.trim());
+
+    if (splitElement.length >= 2) {
+      resultObject[splitElement[0]] = splitElement[1];
```

Comment:
The condition `splitElement.length >= 2` is a good safeguard, but it would be better to explicitly check for the presence of both property and value to avoid adding empty keys or values to the result object.